src/Maestro/Microsoft.DotNet.Maestro.Tasks/src/PushMetadataToBuildAssetRegistry.cs

```diff
@@ -580,6 +580,10 @@ private void LookupForMatchingGitHubRepository(BuildData mergedBuild)
                 {
                     string[] segments = mergedBuild.AzureDevOpsRepository.Split('/');
                     string repoName = segments[segments.Length - 1];
+                    if (repoName.EndsWith("-Trusted", StringComparison.OrdinalIgnoreCase))
```

Comment:
I still think it's important that we only do this if the azdo repo is hosted in devdiv